server/http_status.go

```diff
@@ -103,11 +103,15 @@ func (s *Server) startHTTPServer() {
 		router.Handle("/regions/meta", regionHandler{tikvHandlerTool}).Name("RegionsMeta")
 		router.Handle("/regions/hot", regionHandler{tikvHandlerTool}).Name("RegionHot")
 		router.Handle("/regions/{regionID}", regionHandler{tikvHandlerTool})
+	}
+
+	if s.cfg.Store == "tikv" || s.cfg.Store == "mocktikv" {
 		router.Handle("/mvcc/key/{db}/{table}/{handle}", mvccTxnHandler{tikvHandlerTool, opMvccGetByKey})
 		router.Handle("/mvcc/txn/{startTS}/{db}/{table}", mvccTxnHandler{tikvHandlerTool, opMvccGetByTxn})
```

Comment:
There is some problem when I use this HTTP API when store is `mocktikv`.

Reply:
Comment Addressed, test passed in local with single TiDB + Mocktikv, PTAL